lib/filecheck/src/checker.rs

```diff
@@ -337,23 +338,23 @@ impl<'a> State<'a> {
         } else {
             // We need the captures to define variables.
             rx.captures(txt).map(|caps| {
-                let matched_range = caps.pos(0).expect("whole expression must match");
+                let matched_range = caps.get(0).expect("whole expression must match");
                 for var in defs {
-                    let txtval = caps.name(var).unwrap_or("");
+                    let txtval = caps.name(var).map(|_| "var").unwrap_or("");
```

Comment:
oh. yeah, lol. I meant var. since name() now returns an option of Match.
let me patch this quickly.